pack/Microsoft.NET.Sdk.Publish.Package/Microsoft.NET.Sdk.Publish.nuproj

```diff
@@ -11,7 +11,7 @@
     <IntermediateOutputPath>$(WebSdkRoot)\obj\$(Configuration)\</IntermediateOutputPath>
     <RuntimeIdenfier Condition="'$(RuntimeIdenfier)' == ''">win7-x86</RuntimeIdenfier>
     <CurrentDate Condition="'$(CurrentDate)'==''">-$([System.DateTime]::Now.ToString(yyyyMMdd))</CurrentDate>
-    <WebSdkVersion Condition="'$(WebSdkVersion)' ==''">1.0.0-rel$(CurrentDate)-1</WebSdkVersion>
+    <WebSdkVersion Condition="'$(WebSdkVersion)' ==''">1.0.0-dev$(CurrentDate)-1</WebSdkVersion>
```

Comment:
Should we be referring to `$(BUILD_NUMBER)` and `$(BuildBranchName)` in this one and the one below?